Improve upgrade script and text (#468)
owncloud/docs-server: modules/admin_manual/pages/installation/manual_installation/script_guided_install.adoc

.In case of an upgrade
the old instance path is backed up by _renaming and adding a time stamp_. A new target folder with the old name is created. This ensures that in case of extracting and preparation issues, you can easily go back to the previous version. After a successful upgrade, you must manually remove the backup folder.

TIP: If you do not install or upgrade, the script sets ownership and permissions on linked directories only.

NOTE: In case of upgrading, if you have customized the `.htaccess` or `.user.ini` file in the ownCloud webroot, you have to manually restore any changes made after the script has finished from the backup folder. Take care to only restore the changes made but not the complete file as this file will be recreated on upgrades by the tar source and may contain different settings provided by ownCloud. Customizing `.htaccess` can be necessary when you e.g. xref:configuration/integration/ms-teams.adoc[Integrate ownCloud into Microsoft Teams].

> NOTE: In case of upgrading, if you have used links for the apps-external/ and data/ directory, the script will relink these directories. If you have used standard directories, you manually have to migrate any content of these directories back after the script has run. Check that ownership and permissions are correct after migration and in doubt, let the scrips set the correct ones when using linked directories or ste them manually - which can take a while in any case depending on the size of the data/ directory. Note you have take care not to overwrite new app versions with older ones.

Reviewer: ```suggestion
NOTE: In case of upgrading, if you have created links to the apps-external/ and data/ directory, the script will re-link these directories. If you used standard directories, you have to manually migrate any content of these directories back after the script has run. Check that ownership and permissions are correct after migration and in doubt, let the scrips set the correct ones when using linked directories or set them manually - which can take a while depending on the size of the data/ directory. Take care not to overwrite new app versions with older ones.
```

* **Do you want to chmod/chown these links (y/N)?** +
  This question is only asked when you use links. If you are not installing or upgrading, 
  answering with yes, you can e.g. re-apply ownership and permissions to the linked `data` and
  `apps-external` directories. As written above, the data directory can be very large
  and may take long to complete. Note, by design, there is no progressbar... 

* **Please specify the tar file to extract with full path:** +
  Used when installing or upgrading only. Enter the full path/filename to the tar source
  file, downloaded from ownCloud.

== Final Steps

> When running the script, you will see success messages when finished.

Reviewer: ```suggestion
After running the script, you will see success messages when it finishes.
```

.When installing
Enter the URL of your ownCloud instance in a browser and continue the setup via the graphical installation wizard. For more information, see xref:installation/installation_wizard.adoc[The Installation Wizard].

> The URL you need to enter in the browser depends on the webserver setup taken and either is an external URL or if you installed locally, you should be able to access your ownCloud instance via a browser at `\http://127.0.0.1/owncloud/` or `\http://localhost/owncloud/`.

Reviewer: ```suggestion
The URL you need to enter in the browser depends on the webserver setup and is either an external URL or, if you installed locally, you should be able to access your ownCloud instance via a browser at `\http://127.0.0.1/owncloud/` or `\http://localhost/owncloud/`.
```

.When upgrading

> Follow the steps printed on the screen, especially when using standard directories where you have to migrate the `data/` and `apps-external/` directories manually befor finalizing the upgrade. Do not forget to reapply manually made changes made to `.htaccess` and `.user.ini` in your owncloud root directory.

Reviewer: ```suggestion
Follow the steps printed on the screen, especially when using standard directories where you have to migrate the `data/` and `apps-external/` directories manually before finalizing the upgrade. Do not forget to reapply manually made changes made to `.htaccess` and `.user.ini` in your owncloud root directory.
```

> .How to get the difference of two files quickly

Reviewer: ```suggestion
.How to see the difference between two files quickly
```

> The following example command eases to find the differences of two files, which is helpful for reapplying manually added changes to `.htaccess` and `.user.ini`. Replace that paths, directories and files accordingly.

Reviewer: ```suggestion
The following example command shows you the difference between two files, which is helpful for reapplying manual changes to `.htaccess` and `.user.ini`. Change the paths, directories and files accordingly.
```
